pbnh/tests/paste_curl_test.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python3
+import pycurl
+import unittest
+import json
+import hashlib
+import tempfile
+import os
+
+from pbnh import conf
+from pbnh import app
+
+config = conf.get_config().get('server')
+host = config.get('bind_ip')
+port = config.get('bind_port')
+URL = 'http://{}:{}/'.format(host, port)
+
+
+class TestPost(unittest.TestCase):
+    def setUp(self):
+        self.c = pycurl.Curl()
+        self.c.setopt(pycurl.URL, URL)
+        self.db_fd, app.app.config['DATABASE'] = tempfile.mkstemp()
```

Comment:
Yeah, all of that is fine, it is the explicit database creation that I was wanting. You actually didn't even need to change CreateDB at all as long as you passed the right dbname and driver to it. As an example, check out https://github.com/bhanderson/pbnh/blob/master/pbnh/tests/paste_psql_test.py#L14

You basically just initialize CreateDB with your configuration and then call create. It will sort out all of the rest for you.